app/models/cloud_volume.rb

```diff
@@ -70,9 +70,9 @@ def my_zone
   # instance and a userid are mandatory. Any +options+ are forwarded as
   # arguments to the +create_volume+ method.
   #
-  def self.create_volume_queue(userid, ext_management_system, options = {})
-    task_opts = {
-      :action => "creating Cloud Volume for user #{userid}",
+  def self.create_volume_queue(userid, ext_management_system, options = {}, task_opts = nil)
+    task_opts ||= {
+      :action => "Creating Cloud Volume for user #{userid}",
       :userid => userid
     }
```

Comment:
It would be simpler on the method that overrides if you do the following:
```suggestion
  def self.create_volume_queue(userid, ext_management_system, options = {}, task_opts = {})
    task_opts.reverse_merge!(
      :action => "Creating Cloud Volume for user #{userid}",
      :userid => userid
    )
```

This way they only have to provide the options they want to change, and the defaults will still be maintained without having to be copied